tests/test_arn_s3.py

```diff
@@ -8,13 +8,13 @@ def test_parses_access_point_arn(make_arn):
 
 
 def test_parses_bucket_arn(make_arn):
-    arn = make_arn("s3", "foo")
+    arn = make_arn("s3", "foo", region='', account='')
```

Comment:
```suggestion
    arn = make_arn("s3", "foo", region="", account="")
```
we use https://github.com/psf/black and it requires single quotes. I just updated our CircleCI settings to run on forked PRs, so it should lint properly on the next commit